compiler/src/test/kotlin/dev/zacsweers/metro/compiler/fir/AggregationTest.kt

```diff
@@ -898,6 +890,110 @@ class AggregationTest : MetroCompilerTest() {
     }
   }
 
+  @Test
+  fun `ContributesTo can be repeated to contribute to multiple scopes in a downstream module`() {
+    val previousCompilation =
+      compile(
+        source(
+          """
+          abstract class AltScope private constructor()
+          abstract class ThirdScope private constructor()
+
+          @ContributesTo(AppScope::class)
+          @ContributesTo(AltScope::class)
+          @ContributesTo(ThirdScope::class)
+          interface ContributedInterface {
+            @Provides
+            fun provideValue(): String = "Hello, world!"
+          }
+        """
+            .trimIndent()
+        )
+      )
+
+    compile(
+      source(
+        """
+          @DependencyGraph(scope = AppScope::class)
+          interface ExampleGraph {
+            val myVal: String
+          }
+
+          @DependencyGraph(scope = AltScope::class)
+          interface AltGraph {
+            val altVal: String
+          }
+
+          @DependencyGraph(scope = ThirdScope::class)
+          interface ThirdGraph {
+            val thirdVal: String
+          }
+        """
+          .trimIndent()
+      ),
+      previousCompilationResult = previousCompilation,
+    ) {
+      val appGraphClass = ExampleGraph
+      val appGraph = appGraphClass.generatedMetroGraphClass().createGraphWithNoArgs()
+      appGraphClass.assertHasSupertype("test.ContributedInterface")
+      assertThat(appGraph.callProperty<String>("myVal")).isEqualTo("Hello, world!")
+
+      val altGraphClass = classLoader.loadClass("test.AltGraph")
+      val altGraph = altGraphClass.generatedMetroGraphClass().createGraphWithNoArgs()
+      altGraphClass.assertHasSupertype("test.ContributedInterface")
+      assertThat(altGraph.callProperty<String>("altVal")).isEqualTo("Hello, world!")
+
+      val thirdGraphClass = classLoader.loadClass("test.ThirdGraph")
+      val thirdGraph = thirdGraphClass.generatedMetroGraphClass().createGraphWithNoArgs()
+      thirdGraphClass.assertHasSupertype("test.ContributedInterface")
+      assertThat(thirdGraph.callProperty<String>("thirdVal")).isEqualTo("Hello, world!")
+    }
+  }
+
+  private fun Class<*>.assertHasSupertype(superTypeFqName: String) {
```

Comment:
nit: let's call this `assertHasContributedSupertype` to make it more obvious that this is going to look for the `$$MetroContribution` type